public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionCompleteProgressBar.js

```diff
@@ -71,25 +71,31 @@ function ModalMissionCompleteProgressBar (uiModalMissionComplete) {
      * @private
      */
     this.update = function (missionDistanceRate, userAuditedDistanceRate, otherAuditedDistanceRate) {
+
+        // Round the rates to 0.01 accuracy.
+        var roundedMissionDistanceRate = parseFloat(missionDistanceRate.toFixed(2));
+        var roundedUserAuditedDistanceRate = parseFloat(userAuditedDistanceRate.toFixed(2));
+        var roundedOtherAuditedDistanceRate = parseFloat(otherAuditedDistanceRate.toFixed(2));
+
         horizontalBarOtherContribution.attr("width", 0)
             .transition()
             .delay(200)
             .duration(600)
-            .attr("width", otherAuditedDistanceRate * svgCoverageBarWidth);
+            .attr("width", roundedOtherAuditedDistanceRate * svgCoverageBarWidth);
 
         horizontalBarPreviousContribution.attr("width", 0)
-            .attr("x", otherAuditedDistanceRate * svgCoverageBarWidth)
+            .attr("x", roundedOtherAuditedDistanceRate * svgCoverageBarWidth)
             .transition()
             .delay(800)
             .duration(600)
-            .attr("width", userAuditedDistanceRate * svgCoverageBarWidth);
+            .attr("width", roundedUserAuditedDistanceRate * svgCoverageBarWidth);
 
         horizontalBarMission.attr("width", 0)
-            .attr("x", (otherAuditedDistanceRate + userAuditedDistanceRate) * svgCoverageBarWidth)
+            .attr("x", (roundedOtherAuditedDistanceRate + roundedUserAuditedDistanceRate) * svgCoverageBarWidth)
             .transition()
             .delay(1400)
             .duration(600)
-            .attr("width", missionDistanceRate * svgCoverageBarWidth);
-        horizontalBarMissionLabel.text(parseInt((userAuditedDistanceRate + missionDistanceRate) * 100, 10) + "%");
+            .attr("width", roundedMissionDistanceRate * svgCoverageBarWidth);
```

Comment:
Just a quick idea I had: even if the mission the user did is not worth even 1 pixel, I think we should ensure that they still get a green bar. So I would change this line to
```
.attr("width", Math.max(1, roundedMissionDistanceRate * svgCoverageBarWidth));
```